src/app/app.component.scss

```diff
@@ -4,4 +4,4 @@
 
 ul, li{
   display: inline;
```

Comment:
why ul, li? you want to have all li's under an ul to display inline or all ul's and li's to display inline and why?